server/config/database.js

```diff
@@ -1,4 +1,6 @@
-require('dotenv/config');
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
@yomigeek you've a nice work in completing this task. 🙌🏾
I would like to know, though, your reasoning behind this import modification because I feel you basically did the same thing in a longer way.